loop-templates/templates/src/index.ts.squirrelly

```diff
@@ -34,6 +37,9 @@ import {
 {{ @if (it.aptitudes.includes('clipboard')) }}
   clipboardListener.listen();
 {{ /if }}
+{{ @if (it.aptitudes.includes('cursor')) }}
+  cursorListener();
```

Comment:
Another bit for consistency, can we try changing this to be `cursorListener.listen()`?